test/cs_guide/resources/resources_test.exs

```diff
@@ -2,200 +2,199 @@ defmodule CsGuide.ResourcesTest do
   use CsGuide.DataCase
 
   alias CsGuide.Resources
+  alias CsGuide.Resources.Brand
+  alias CsGuide.Fixtures
+
+  @create_brand Fixtures.create_brand()
 
   describe "venues" do
     alias CsGuide.Resources.Venue
 
-    @valid_attrs %{phone_number: "some phone_number", postcode: "some postcode", venue_name: "some venue_name"}
-    @update_attrs %{phone_number: "some updated phone_number", postcode: "some updated postcode", venue_name: "some updated venue_name"}
+    @valid_attrs %{
+      phone_number: "some phone_number",
+      postcode: "some postcode",
```

Comment:
TODO (_post launch_): `postcode` should use `Fields.postcode` to automatically perform data validation. 
see: https://github.com/dwyl/fields/issues/18 (_create the field in the **reusable** module then **use** it here, that way the maintenance is done centrally and all "Client Apps" benefit from improvements._) 👍